src/tesseract.jl

```diff
@@ -40,6 +40,8 @@ Errors / Warnings are reported through `Logging`, so no exceptions are thrown.
 # Arguments
 - `input_path::String`: Path to the image to be processed
 - `output_path::String`: Path to the text result to be written
+- `extra_args` (`String`s): Optional arguments to change the nature of the output
```

Comment:
Just to learn about docstrings practices: it is ok to label this argument as this way instead of `extra_args::String...`?

Reply:
Mine was more informal, yours is real Julia syntax. Let's go with your version.